app/frontend/walletApp.js

```diff
@@ -51,6 +53,25 @@ window.onhashchange = () =>
     },
   })
 
+Sentry.init({
+  dsn: 'https://d77d3bf9d9364597badab9c00fa59a31@sentry.io/1501383',
```

Comment:
don't forget to put this in some kind of config

Reply:
Again I was inspired by myEtherwallet, but it definitely could be in .env and let it be added by Heroku but it didn't seem necessary.